node/actors/bft/src/tests/unit_tests.rs

```diff
@@ -0,0 +1,599 @@
+/// - [x] replica_prepare_old_view
+/// - [x] replica_prepare_during_commit
+/// - [ ] replica_prepare_not_leader_in_view
+/// - [ ] replica_prepare_already_exists
+/// - [ ] replica_prepare_invalid_sig
+/// - [ ] replica_prepare_invalid_commit_qc_different_views
+/// - [ ] replica_prepare_invalid_commit_qc_signers_list_empty
+/// - [ ] replica_prepare_invalid_commit_qc_signers_list_invalid_size
+/// - [ ] replica_prepare_invalid_commit_qc_multi_msg_per_signer
+/// - [ ] replica_prepare_invalid_commit_qc_insufficient_signers
+/// - [ ] replica_prepare_invalid_commit_qc_invalid_aggregate_sig
+/// - [ ] replica_prepare_high_qc_of_future_view
+/// - [x] replica_prepare_num_received_below_threshold
+/// -
+/// - [x] leader_prepare_sanity
+/// - [ ] leader_prepare_sanity_yield_replica_commit
+/// - [x] leader_prepare_invalid_leader
+/// - [x] leader_prepare_old_view
+/// - [x] leader_prepare_invalid_sig
+/// - [x] leader_prepare_invalid_prepare_qc_different_views
+/// - [ ] leader_prepare_invalid_prepare_qc_signers_list_empty
+/// - [ ] leader_prepare_invalid_prepare_qc_signers_list_invalid_size
+/// - [ ] leader_prepare_invalid_prepare_qc_multi_msg_per_signer
+/// - [ ] leader_prepare_invalid_prepare_qc_insufficient_signers
+/// - [ ] leader_prepare_invalid_prepare_qc_invalid_aggregate_sig
+/// - [ ] leader_prepare_high_qc_of_future_view
+/// - [ ] leader_prepare_proposal_oversized_payload
+/// - [ ] leader_prepare_proposal_mismatched_payload
+/// - [ ] leader_prepare_proposal_when_previous_not_finalized
+/// - [ ] leader_prepare_proposal_invalid_parent_hash
+/// - [ ] leader_prepare_proposal_non_sequential_number
+/// - [ ] leader_prepare_reproposal_without_quorum
+/// - [ ] leader_prepare_reproposal_when_finalized
+/// - [ ] leader_prepare_reproposal_invalid_block
+/// - [ ] leader_prepare_yield_replica_commit
+/// -
+/// - [ ] replica_commit_sanity
+/// - [ ] replica_commit_sanity_yield_leader_commit
+/// - [ ] replica_commit_old
+/// - [ ] replica_commit_not_leader_in_view
+/// - [ ] replica_commit_already_exists
+/// - [ ] replica_commit_invalid_sig
+/// - [ ] replica_commit_unexpected_proposal
+/// - [ ] replica_commit_num_received_below_threshold
+/// - [ ] replica_commit_make_commit_qc_failure_distinct_messages
+/// -
+/// - [ ] leader_commit_sanity
+/// - [ ] leader_commit_sanity_yield_replica_prepare
+/// - [ ] leader_commit_invalid_leader
+/// - [ ] leader_commit_old
+/// - [ ] leader_commit_invalid_sig
+/// - [ ] leader_commit_invalid_commit_qc_signers_list_empty
+/// - [ ] leader_commit_invalid_commit_qc_signers_list_invalid_size
+/// - [ ] leader_commit_invalid_commit_qc_insufficient_signers
+/// - [ ] leader_commit_invalid_commit_qc_invalid_aggregate_sig
+///
+#[tokio::test]
+async fn replica_prepare_sanity() {
+    let mut util = Util::make().await;
+
+    let replica_prepare = util.make_replica_prepare(None::<fn(&mut ReplicaPrepare)>);
+    let res = util.dispatch_replica_prepare(replica_prepare);
+    assert_eq!(res, Ok(()));
```

Comment:
In this case, `unwrap()`ing the result would be enough to check it.

Reply:
or `assert!(res.is_ok())`.

Reply:
Fixed.